plugins/modules/mso_tenant.py

```diff
@@ -6,13 +6,12 @@
 # GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
+
 __metaclass__ = type
 
-ANSIBLE_METADATA = {'metadata_version': '1.1',
-                    'status': ['preview'],
-                    'supported_by': 'community'}
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
 
-DOCUMENTATION = r'''
+DOCUMENTATION = r"""
```

Comment:
I see a lot of difference usage of double vs single quotes throughout the other modules. What is our defined standard for this?

Reply:
I am using **black** Python code formatter to align the code. It would be better if we use **black** code formatter.

FYI: https://pypi.org/project/black/

Reply:
Ok will we use this then on the entire project directory? 
Or just when we make changes to a module and or piece of code?

Reply:
We can use it for the entire project directory. So that we can avoid formatting errors.